### ISSUE TITLE
FEATURE REQUEST: Add Google/Apple sign in on New Dot, like we have on Old Dot

### ISSUE THREAD

**Issue by mvtglobally** [Expensify/App#7079]
Design doc: https://docs.google.com/document/d/1W0hCQbcvcFIH1mlbSElovXKNaP_FK9J3MU3ESzH-y34/edit#

Pre-design: https://expensify.slack.com/archives/C01GTK53T8Q/p1678126175371849

Technical pre-design: https://expensify.slack.com/archives/C01GTK53T8Q/p1680016894091629

If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open app 
2. Try to login with Google/Apple sign in

## Expected Result:
User should be able to login with Google/Apple sign in 

## Actual Result:
User can only login with email 

## Workaround:
Unknown 

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.1.26-0 
**Reproducible in staging?:** Y 
**Reproducible in production?:** Y 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation 
![Screen Shot 2022-01-07 at 12 52 26 AM](https://user-images.githubusercontent.com/43995119/148498557-7004d73e-79ac-40e9-baca-b136b31851e6.png)
![Screen Shot 2022-01-07 at 12 52 14 AM](https://user-images.githubusercontent.com/43995119/148498562-fa8c7410-7589-47b1-bbf2-f7e7ba64e636.png)

**Expensify/Expensify Issue URL:**
**Issue reported by:** @puneetlath  
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1641509570059600 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @alexpensify (`AutoAssignerTriage`), see https://stackoverflow.com/c/expensify/questions/4749 for more details.

**Comment by MelvinBot** [comment]
Triggered auto assignment to @marcochavezf (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by marcochavezf** [comment]
This issue should be internal because it will require checking the implementation of Google/Apple of OldDot to make it compatible with the corresponding Web-E commands.

**Comment by MelvinBot** [comment]
Triggered auto assignment to @puneetlath (`Internal`) because issue was reported by a contributor who needs to be compensated if this issue is fixed

**Comment by marcochavezf** [comment]
Update: I did a bit of research of the most maintained third-party libraries for both Google and Apple sign-in functionality and those are:
- https://github.com/react-native-google-signin/google-signin
- https://github.com/invertase/react-native-apple-authentication

Also asked in [#expensify-open-source](https://expensify.slack.com/archives/C01GTK53T8Q/p1641859408263000) for a second opinion and we're ok using those libraries.

**Comment by puneetlath** [comment]
Would we also use libraries for web/desktop? Or just do it ourselves?

**Comment by marcochavezf** [comment]
> Would we also use libraries for web/desktop? Or just do it ourselves?

Ah that's a good question, I don't know yet, also I'm not sure if we can re-use the native libraries on web/desktop. I will investigate and compare.

**Comment by marcochavezf** [comment]
Related to using libraries for web/desktop, TL:DR; I found out for Google SignIn is better to implement it ourselves. 

I checked out the implementation of https://github.com/anthonyjgrove/react-google-login (which is the most popular library for Google SignIn in regular React) and I found out the implementation is relatively simple and it's better to implement it ourselves because the library contains features that we don't need (like a specific Google button, offline access, Google SignOut) and we should adapt our code to the library (create a new component to wrap the library) when we can create a function and simply hook it up to a regular Button component.

**Comment by marcochavezf** [comment]
When I'm testing the Google Sign-in I'm having the error `Error 403: restricted_client` on iOS because the OAuth consent screen is not set up in Google Cloud Platform:

<img width="332" alt="Screen Shot 2022-02-07 at 19 01 32" src="https://user-images.githubusercontent.com/6829422/153052438-7bf039a6-654e-4209-8ce0-53b32f1f46b0.png">

Rory (`ring3`) was trying to help with that but he doesn't have permission. I'm going to need someone of `ring0` to set it up in GCP. These are the instructions:

1. Open https://console.developers.google.com/apis/credentials?project=expensify-chat and click on `CONFIGURE CONSENT SCREEN`:

<img width="800" alt="Screen Shot 2022-02-08 at 12 30 59" src="https://user-images.githubusercontent.com/6829422/153053641-e68cf182-5768-4716-8cca-38514e2c6696.png">

2. Select `External` and `CREATE` since the Google Sign-in in NewDot is for any user with a Google Account:

<img width="800" alt="Screen Shot 2022-02-08 at 12 34 01" src="https://user-images.githubusercontent.com/6829422/153053680-f6107d3c-30f5-41ec-8a1a-12ca577fc7de.png">

3. I think that should be it, I don't know what's beyond after clicking on `CREATE` 😅😂

**Comment by marcochavezf** [comment]
When I'm testing the Google Sign-in I'm having the error `Error 403: restricted_client` on iOS because the OAuth consent screen is not set up in Google Cloud Platform:

<img width="300" alt="Screen Shot 2022-02-07 at 19 01 32" src="https://user-images.githubusercontent.com/6829422/153052438-7bf039a6-654e-4209-8ce0-53b32f1f46b0.png">

Rory (`ring3`) was trying to help with that but he also doesn't have permission. I'm going to need someone of `ring0` to set it up in GCP.

**Comment by rafecolton** [comment]
Did my best to configure the consent screen but not sure if it's correct, and it says it may need 4-6 weeks for Google to verify before we can publish it.

**Comment by marcochavezf** [comment]
Update: 
- The Google SignIn has been implemented for iOS and Android and it's working fine locally.
- I implemented the Google SignIn on Desktop using the google-auth-library-nodejs library. I tried using first the same approach used on web, but I was having issues with Electron (https://expensify.slack.com/archives/C01GTK53T8Q/p1644432240338919). Also, I found out the web library hasn't been officially supported for Electron ([link 1](https://github.com/google/google-api-javascript-client/issues/222#issuecomment-368565707) and [link 2](https://github.com/google/google-api-javascript-client/blob/master/docs/start.md#supported-environments)) and there have situations where all Electron apps using the web library suddenly had [issues]((https://github.com/google/google-api-javascript-client/issues/506) and needed to wait until Electron released an update.

**Comment by marcochavezf** [comment]
No update this week, I've been focused on an N7 issue also I'm going to be OOO for a few days. I will come back to it next week.

**Comment by marcochavezf** [comment]
No update this week, coming from OOO. One pending item is the configuration of authorized javascript origins in GCP for web. Seems it's safe ([slack convo](https://expensify.slack.com/archives/C03TQ48KC/p1645227154551739)) to include localhost as authorized origin to test Google Sign-in locally because we're only retrieving the email user from the Google API (non-sensitive scope).

**Comment by marcochavezf** [comment]
These are the new sign-in buttons with an error message:

<img width="800" alt="Screen Shot 2022-02-04 at 17 55 30" src="https://user-images.githubusercontent.com/6829422/159363296-d3b42dc3-2b52-44a5-a3f9-50262857a3ed.png">

<img width="400" alt="Screen Shot 2022-02-04 at 17 56 01" src="https://user-images.githubusercontent.com/6829422/159363300-54f48e37-e6c5-4833-90e1-3667c5e2ed90.png">

And this is how the buttons look when the user clicks on an option (Google Sign-in in this case) and it's loading:

<img width="400" alt="Screen Shot 2022-02-04 at 17 56 01" src="https://user-images.githubusercontent.com/6829422/159364907-590f25be-15b4-48a6-9c6a-37baff7d4f90.png">

**Comment by marcochavezf** [comment]
Hi @michelle-thompson! In the above message I posted a few screenshots of a new page that contains the new sign-in buttons, do you think the style for those buttons is fine? :)

**Comment by marcochavezf** [comment]
No update yet, I'm focusing on App Offline First atm

**Comment by marcochavezf** [comment]
No update yet, focused on App Offline First, but I want to continue with this issue soon

**Comment by marcochavezf** [comment]
No update yet, working on some backlog from past weeks

**Comment by marcochavezf** [comment]
No update, still working on some backlog. Also currently in NewDot, we require passwords for new accounts, and according to the N7 onboarding pre-design meeting, seems [new members won't need to set a password](https://expensify.slack.com/archives/C02HWMSMZEC/p1651175927632839). I think it will be a good idea to coordinate the detailed implementation of N7 onboarding with the SSO implementation. I'm going to change the priority to monthly at the moment.

**Comment by marcochavezf** [comment]
No update yet

**Comment by marcochavezf** [comment]
I'm going to put this issue on hold because the implementation will need to be adapted to the [Authentication API refactoring](https://github.com/Expensify/Expensify/issues/212820) and also should be coordinated with the N7 Onboarding implementation.

**Comment by marcochavezf** [comment]
No update

**Comment by marcochavezf** [comment]
No update

**Comment by marcochavezf** [comment]
No update, focused on WAQ

**Comment by JmillsExpensify** [comment]
Clarified the WAQ hold and added myself as an assignee to help with when we pick this back up.

**Comment by JmillsExpensify** [comment]
Still have a long way to go before we get back to this.

**Comment by JmillsExpensify** [comment]
As WAQ starts to wind down, we should decide where this optimally fits. I don't have a particularly strong view, though we really don't require it until N8 at the earliest. At that point it'd be possible to sign up on NewDot for the first time, including getting started with a Free Plan. Obviously N10 at the latest.

**Comment by puneetlath** [comment]
N8 feels like a good spot to me.

**Comment by JmillsExpensify** [comment]
Cool, added to that project for the time being.

**Comment by JmillsExpensify** [comment]
No longer on hold for WAQ, so updating the title.

**Comment by roryabraham** [comment]
Should we make this a weekly?

**Comment by shawnborton** [comment]
Not overdue

**Comment by JmillsExpensify** [comment]
Awesome that sounds great! It'd be good to start taking on this level of polish, and we'll need to do it soon enough anyway.

**Comment by marcochavezf** [comment]
@JmillsExpensify @roryabraham @puneetlath I drafted this [P/S](https://docs.google.com/document/d/1NV_ctFdchqXKa4PXf-43Dh3_Uk9oXKRhEMOBNBZccvo/edit) to use the "Expert Contributor buddy process" to implementation of the Google and Apple sign-in/sign-up. I'll appreciate any feedback you can give 🙇🏽

**Comment by JmillsExpensify** [comment]
Thanks! Left a couple of comments in the doc.

**Comment by puneetlath** [comment]
Ship it!

**Comment by marcochavezf** [comment]
P/S [posted](https://expensify.slack.com/archives/CC7NECV4L/p1677005286500849) in #whatsnext, thank you for the feedback!

**Comment by marcochavezf** [comment]
Update: we're going to be working with [Infinite Red](https://expensify.slack.com/archives/C04KQL2HSQ1/p1677804785571899) to implement the Google/Apple sign-up/sign-in. We're going to do a pre-design meeting on Monday.

**Comment by MelvinBot** [comment]
:wave: Hello Generalist Track Team - you have been assigned to review this **High Level** Design Doc. Please, treat this as a priority. Review [this Stack Overflow](https://stackoverflow.com/c/expensify/questions/5838) for some tips on reviewing a design doc. Once you are done, simply press the `Add "Reviewed Doc" comment` button in the right hand side K2 pannel or follow [these](https://stackoverflow.com/c/expensify/questions/13713) instructions.

---
- `@mlouiser` (Expensifier / Graduate) - https://github.com/Expensify/App/issues/15905
- `@johncschuster` (Expensifier / Graduate) - https://github.com/Expensify/App/issues/15906
- `@mateocole` (Project Manager) - https://github.com/Expensify/App/issues/15907
- `@zsgreenwald` (Project Manager) - https://github.com/Expensify/App/issues/15908
- `@isabelastisser` (Product Manager) - https://github.com/Expensify/App/issues/15909
- `@stephanieelliott` (Product Manager) - https://github.com/Expensify/App/issues/15910
- `@joaniew` (Generalist) - https://github.com/Expensify/App/issues/15911
- `@quinthar` (Generalist) - https://github.com/Expensify/App/issues/15912
- `@kortneywynn` (Accounting Technical Team) - https://github.com/Expensify/App/issues/15913

**Comment by kortneywynn** [comment]
I have read and reviewed this Design Doc! Please see accounting comments within.

**Comment by quinthar** [comment]
I have read and reviewed this Design Doc!

**Comment by JmillsExpensify** [comment]
I have read and reviewed this Design Doc!

**Comment by jliexpensify** [comment]
I have read and reviewed this Design Doc!

**Comment by isabelastisser** [comment]
I have read and reviewed this Design Doc!

**Comment by dangrous** [comment]
I have read and reviewed this Design Doc!

**Comment by stephanieelliott** [comment]
I have read and reviewed this Design Doc!

**Comment by shawnborton** [comment]
Not overdue

**Comment by trjExpensify** [comment]
I have read and reviewed this Design Doc!

**Comment by johncschuster** [comment]
I have read and reviewed this Design Doc!

**Comment by johncschuster** [comment]
I have read and reviewed this Design Doc!

**Comment by johncschuster** [comment]
I have read and reviewed this Design Doc!

**Comment by mlouiser** [comment]
I have read and reviewed this Design Doc!

**Comment by marcochavezf** [comment]
The [detailed section](https://docs.google.com/document/d/1W0hCQbcvcFIH1mlbSElovXKNaP_FK9J3MU3ESzH-y34/edit#bookmark=id.w7400olmxs5y) is ready for review. Applying the label design doc label again.

**Comment by tgolen** [comment]
I have read and reviewed this Design Doc!

**Comment by joaniew** [comment]
b

**Comment by joaniew** [comment]
I have read and reviewed this Design Doc!

**Comment by AndrewGable** [comment]
I have read and reviewed this Design Doc!

**Comment by JmillsExpensify** [comment]
I have read and reviewed this Design Doc!

**Comment by ryanschaffer** [comment]
I have read and reviewed this Design Doc!

**Comment by cjjohnson121** [comment]
I have read and reviewed this Design Doc!

**Comment by Beamanator** [comment]
I have read and reviewed this Design Doc!

**Comment by flodnv** [comment]
I have read and reviewed this Design Doc!

**Comment by coleaeason** [comment]
I have read and reviewed this Design Doc!

**Comment by stitesExpensify** [comment]
I have read and reviewed this Design Doc!

**Comment by stephanieelliott** [comment]
I have read and reviewed this Design Doc!

**Comment by pecanoro** [comment]
I have read and reviewed this Design Doc!

**Comment by anmurali** [comment]
I have read and reviewed this Design Doc!

**Comment by ryanldonato** [comment]
I have read and reviewed this Design Doc!

**Comment by lindboe** [comment]
Hi @shawnborton ! We discussed wanting to mock up some screen designs for the "landing page" for Desktop sign-in. [I've written up what this entails in the design doc](https://docs.google.com/document/d/1W0hCQbcvcFIH1mlbSElovXKNaP_FK9J3MU3ESzH-y34/edit#heading=h.xotc9qdxdbh2) along with a user flow diagram, where the bottom lane represents how I would categorize the states this page can be in. Below the diagram I also wrote out two flows in more detail (one happy path, one error) that should hopefully be easier to follow since they're linear. Let me know if that's not enough info and I can get you more!

There's still two outstanding details I put up for discussion that would affect the user experience in this flow, but shouldn't significantly affect the design of this screen, aside from some copy details:
* if the user doesn't opt to use their existing web session on desktop and continues with third-party sign-in, we may need to log them out of their existing web session ([Slack link](https://expensify.slack.com/archives/C01GTK53T8Q/p1683067753346469?thread_ts=1682551167.897329&cid=C01GTK53T8Q))
* if an error and the user goes "back" to the main login form on web, we don't have a good way of prompting them to move to desktop once they're signed in. I asked if we could move the "open in desktop" prompts to only happen once the user is signed in ([Slack link](https://expensify.slack.com/archives/C01GTK53T8Q/p1683070404415199)).

**Comment by shawnborton** [comment]
Okay, here's a few mockups to get us started.

Happy path:
![image](https://user-images.githubusercontent.com/2319350/236012212-72fa2749-8526-4090-8a55-2ac4ee660eb2.png)

![image](https://user-images.githubusercontent.com/2319350/236012222-80937183-87c0-450a-8a01-2f08ff614d72.png)


Already signed in:
![image](https://user-images.githubusercontent.com/2319350/236012263-2b51f349-bd9f-4487-ba07-7c83914ed46e.png)


Error path:
![image](https://user-images.githubusercontent.com/2319350/236012280-b27e8340-96b4-4fe2-8e1e-ed6d336e0a54.png)


Figma file is here: https://www.figma.com/file/pLyF0knXQdbHflO4ILhssB/Google-and-Apple-Sign-In?node-id=1%3A13261&t=We8W1hhO5jXobrcD-1

**Comment by marcochavezf** [comment]
Hi @davidcardoza, can you review the following proposed copy?

> You're logged in as <account>. Click "Open link" in the prompt to log into the desktop app with this account.
> Don't see the prompt? <Try again>, or <continue to the web app>.

This will be shown when the user wants to sign-in with Apple/Google from desktop but it's redirected to web to complete the sign-in, more context [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1688079113495829?thread_ts=1687552578.360189&cid=C01GTK53T8Q). 

cc @lindboe

**Comment by davidcardoza** [comment]
>You're logged in as . Click "Open link" in the prompt to log into the desktop app with this account.
Don't see the prompt? [Try again](https://github.com/Expensify/App/issues/7079), or [continue to the web app](https://github.com/Expensify/App/issues/7079)

Will we be redirecting the user to the desktop app or will the user already be on the desktop app when they receive this message? @marcochavezf

**Comment by lindboe** [comment]
> Will we be redirecting the user to the desktop app or will the user already be on the desktop app when they receive this message? 

We will be showing a prompt, whose content is determined by the web browser, to open the desktop app when this is copy is shown underneath. Comparable to this example from Zoom:

<img width="863" alt="Screenshot 2023-07-01 at 3 02 21 PM" src="https://github.com/Expensify/App/assets/5252268/46c1419d-5d6b-4a6f-a313-b57e22a42dab">

Different browsers also show different "Accept" text in their prompts. The example above was in chrome, below there's an example for Firefox (which uses "Open link", instead of naming the application). The zoom example updates the copy accordingly, we could do that if we want, or just use copy that applies to all of them:

<img width="712" alt="Screenshot 2023-07-01 at 3 02 58 PM" src="https://github.com/Expensify/App/assets/5252268/ddcd04b1-8351-4abf-8bdc-baf9da9c340e">

**Comment by davidcardoza** [comment]
Yeah I think we can just use generalized copy that applies to them all. The suggested copy from @marcochavezf looks great.

**Comment by marcochavezf** [comment]
Apple sign-in [PR](https://github.com/Expensify/App/pull/16723) is still in review. Some changes were requested, but besides that the PR looks good to me (already tested on native), so I think we can merge after the changes are addressed.

**Comment by JmillsExpensify** [comment]
Disregard the project "closed" update above. I'm just moving things around external to organize the product roadmap.

**Comment by iwiznia** [comment]
Reverted this since https://github.com/Expensify/App/issues/23285

**Comment by marcochavezf** [comment]
This is not done yet, the Apple sign-in PR was reverted. Latest update [here](https://expensify.slack.com/archives/C04KQL2HSQ1/p1690501483836319). Updating the title and removing the `Awaiting Payment` label.

**Comment by shawnborton** [comment]
Not overdue, see @marcochavezf's post above.

**Comment by marcochavezf** [comment]
Waiting for the new PR

**Comment by marcochavezf** [comment]
There's a draft PR up

**Comment by marcochavezf** [comment]
Hi @cdanwards, do you have an ETA for the PR to be ready for review?

**Comment by marcochavezf** [comment]
Assigning @0xmiroslav as C+ reviewer

**Comment by cdanwards** [comment]
After merging the feature branch, I preemptively opened the tracking issue for the `Cross-Origin-Opener-Policy` Google console error [here](https://github.com/Expensify/App/issues/25427).

Discussion to open this issue can be found [here](https://github.com/Expensify/App/pull/23673#issuecomment-1674742129)

**Comment by lindboe** [comment]
It was noted Google Sign-In isn't working on ad hoc builds: https://expensify.slack.com/archives/C01GTK53T8Q/p1692302447001879

It looks like logging isn't quite getting all the details we'd want. I'm creating a PR to quickly address that.

**Comment by cdanwards** [comment]
Issue found in staging: https://github.com/Expensify/App/issues/25679
PR for Issue: https://github.com/Expensify/App/pull/25706

**Comment by lindboe** [comment]
Bug fix summary:

Blockers:
* Fix passing idToken for Apple sign-in, PR open: https://github.com/Expensify/App/pull/25710
* Fix desktop freeze after redirecting back to desktop app, PR open: https://github.com/Expensify/App/issues/25679
* Google iframe showing due to dark mode, PR fixed: https://github.com/Expensify/App/pull/25645

Non-blockers:
* Improve Google Sign In logging, PR open: https://github.com/Expensify/App/pull/25446
* Content-origin-opener-policy falsely reported error from Google, issue: (https://github.com/Expensify/App/issues/25427)

**Comment by JmillsExpensify** [comment]
Looks like we're just down to two linked PRs above and then we can close this one out.

**Comment by situchan** [comment]
C+ payment for [PR](https://github.com/Expensify/App/pull/23673) review is pending

**Comment by JmillsExpensify** [comment]
Still curious what else is left for this project? Can we pay C+ and close this out?

**Comment by lindboe** [comment]
Hi @JmillsExpensify, I'm not sure of the usual process, but I can list what I'm aware of. @marcochavezf can you add any other context needed?

Tasks: We (Infinite Red) agreed to write a blog post on the design and implementation of Google/Apple sign-in cross-platform to contribute to the React Native community. This has been de-prioritized by other projects, but as far as I'm aware it's the last remaining item to wrap up the project itself.

Bugs:
* Stray console warning from Google sign-in library (for context, this is Google's fault, it's giving a false positive warning about configuration): https://github.com/Expensify/App/issues/25427
* Able to use Sign in with Google to sign in with a locked account: https://github.com/Expensify/App/issues/25713

Outstanding PRs: None now. Jules got [this follow-up PR](https://github.com/Expensify/App/pull/25446) merged for improved error logging for Google Sign-In.

**Comment by marcochavezf** [comment]
Ah yes, I've been swamped with other priorities, apologies for the delay here. I think the bugs that @lindboe posted are more like polish items since are not impacting the critical flow. So we can close this GH and pay @0xmiroslav and @situchan for the C+ review. Also, one pending item is the project wrap-up email, which I can send at the EOW

**Comment by JmillsExpensify** [comment]
As for the project wrap-up though, typically we merge all PRs before sending out the wrap-up, including issue that have a low priority. So mainly I'd make sure that you have a plan for getting those addressed.

**Comment by JmillsExpensify** [comment]
@situchan I've just sent you an offer on Upwork. 

@0xmiroslav I believe your preference is to still hold payments until you indicate you're ready. Just let us know.

**Comment by JmillsExpensify** [comment]
Alright, everyone that can be paid right now has been paid out. @marcochavezf can you confirm that a full set of regression tests have already been added for this project? If so, then all we lack are those two polish items and the wrap-up email.

**Comment by marcochavezf** [comment]
Yes, the regression tests were added to TR [here](https://github.com/Expensify/Expensify/issues/313998)

**Comment by marcochavezf** [comment]
No update, working on other higher priority items, but planning to come back to this one this week

**Comment by marcochavezf** [comment]
No update, wrapping up other higher priority items

**Comment by marcochavezf** [comment]
No update

**Comment by shawnborton** [comment]
What's the latest here?

**Comment by marcochavezf** [comment]
No update

**Comment by JmillsExpensify** [comment]
Any update?

**Comment by JmillsExpensify** [comment]
Still not sure where we're at.

**Comment by JmillsExpensify** [comment]
Same

**Comment by marcochavezf** [comment]
No update

**Comment by JmillsExpensify** [comment]
@marcochavezf What's the latest on this one?

**Comment by marcochavezf** [comment]
No update here since I've been prioritizing wave7 issues. I think the two remaining issues can be part of #vip-vbs

**Comment by quinthar** [comment]
Don't we already have this?
<img width="1440" alt="image" src="https://github.com/Expensify/App/assets/575186/4b7e4ce1-fa2e-4f43-a815-952d0dd90fef">

**Comment by shawnborton** [comment]
Friendly bump - what's the latest on this one @marcochavezf ?

**Comment by marcochavezf** [comment]
Just the wrap-up email is pending but I've been deprioritizing because of wave bugs. I will prioritize it this week for real, sorry for the delay here.

**Comment by marcochavezf** [comment]
Writing the wrap-up email, aiming to send it out later today

**Comment by marcochavezf** [comment]
Email sent, closing it out
